src/Components/Divider.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {AbsoluteFill} from 'remotion';
+
+export const Divider: React.FC<{
+	style: React.CSSProperties;
+	width?: string | number;
+	height?: number;
```

Comment:
I'm not sure if the height is a value that will change, so does it need to be a props ?

Reply:
It doesn't, i've removed it